spaces/S000064/properties/P000110.md

```diff
@@ -0,0 +1,10 @@
+---
+space: S000064
+property: P000110
+value: true
+---
+
+Fix an enumeration $D=\{q_n:n<\omega\}$ and denote $D_n=\{q_k: k\leq n\}$.
+
+A development for $X$ can be formed out of the open covers
+$\mathscr V_n=\{ U_n(x)\setminus D_n : x\in X\setminus D\}\cup\{\{q_k\}:k<\omega \}$, were $U_n(x):=\{x\}\cup\{q\in D:\|q-x\|<\frac1n\}$.
```

Comment:
```suggestion
$\mathscr U_n=\{ U_n(x)\setminus D_n : x\in X\setminus D\}\cup\{\{q\}:q\in D \}$,
where $U_n(x):=\{x\}\cup\{q\in D:\|q-x\|<\frac1n\}$  and $\|.\|$ is the Euclidean norm.
```
$\mathscr U_n$ to match the pi-base definition of developable;
simpler second part of formula for $\mathscr U_n$;
"where" typo;
norm notation.